packages/algob/src/internal/deployer_cfg.ts

```diff
@@ -45,9 +33,9 @@ export class DeployerConfig {
 
   constructor (runtimeEnv: AlgobRuntimeEnv, algoOp: AlgoOperator) {
     this.runtimeEnv = runtimeEnv;
-    this.cpData = loadCheckpointsRecursive();
+    this.cpData = new CheckpointRepoImpl();
```

Comment:
why are we using  `CheckpointRepoImpl` instead of `loadCheckpointsRecursive` ? are these same ?

Reply:
Since we are initializing a new deployer, i was initializing with a fresh cp data (empty). But lets keep `loadCheckpointsRecursive` as user may deploy multiple scripts one by one. (Reverting).

Reply:
yes